[2024-08-26T08:01:28.160+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: json_FT_processing_pipeline_03.bulk_load_task scheduled__2024-08-25T08:00:00+00:00 [queued]>
[2024-08-26T08:01:28.168+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: json_FT_processing_pipeline_03.bulk_load_task scheduled__2024-08-25T08:00:00+00:00 [queued]>
[2024-08-26T08:01:28.168+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 61
[2024-08-26T08:01:28.182+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): bulk_load_task> on 2024-08-25 08:00:00+00:00
[2024-08-26T08:01:28.188+0000] {standard_task_runner.py:60} INFO - Started process 57704 to run task
[2024-08-26T08:01:28.191+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'json_FT_processing_pipeline_03', 'bulk_load_task', 'scheduled__2024-08-25T08:00:00+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/FranceTravail_pipeline_dag03.py', '--cfg-path', '/tmp/tmpaef3esjo']
[2024-08-26T08:01:28.192+0000] {standard_task_runner.py:88} INFO - Job 68: Subtask bulk_load_task
[2024-08-26T08:01:28.205+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-26T08:01:28.238+0000] {task_command.py:423} INFO - Running <TaskInstance: json_FT_processing_pipeline_03.bulk_load_task scheduled__2024-08-25T08:00:00+00:00 [running]> on host 9e648299db30
[2024-08-26T08:01:28.310+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='json_FT_processing_pipeline_03' AIRFLOW_CTX_TASK_ID='bulk_load_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-25T08:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-25T08:00:00+00:00'
[2024-08-26T08:01:33.432+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-26T08:01:33.444+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=json_FT_processing_pipeline_03, task_id=bulk_load_task, execution_date=20240825T080000, start_date=20240826T080128, end_date=20240826T080133
[2024-08-26T08:01:33.464+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-26T08:01:33.479+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
